### ISSUE TITLE
Infra overview no longer current, needs updating

### ISSUE THREAD

**Issue by sindarina** [hachyderm/community#174]
The current infra overview is no longer current, and needs an update;
https://github.com/hachyderm/community/tree/main/infra

**Comment by sindarina** [comment]
Yeah, it probably just needs a redirect to the new community website or whatever, it's a matter of cleaning up after the recent changes. I'm aware of the blog post, this issue is just to notify the admins of conflicting information that might cause confusion down the line.

**Comment by slamp** [comment]
Noted.
I'm thinking of 2 solutions:
A/ update the https://github.com/hachyderm/community/blob/main/infra/README.md and put a link to the blog https://community.hachyderm.io/blog](https://community.hachyderm.io/blog/
B/ Do a quick summary directly in the README.md 

Pros/Cons
A/ quick-win but end-users will have to go thru the blog posts to find the information
B/ End-users will have directly the information but we will have a double maintenance issue 

WDYT ?

**Comment by quintessence** [comment]
👋  Mod here

We're in the process of migrating all of our documentation to the community.hachyderm.io site. Public facing information about the infrastructure will be updated as part of the migration :)

**Comment by quintessence** [comment]
To quickly address the specific pro/con requested: the infra wouldn't live on the blog in this case, but on the docs page.
